templates/first-snap/install-snapcraft.html

```diff
@@ -8,28 +8,22 @@ <h4>What operating system are you developing on?</h4>
   </div>
   <div class="row">
     <div class="col-small-2 col-medium-2 col-2 col-logo">
-      <span class="p-logo-link p-card u-align-text--center js-os-select" data-os="linux">
-        <header class="p-card__header">
-          <img src="https://assets.ubuntu.com/v1/e95bac0c-tux.svg" alt="">
-        </header>
+      <div href="#" class="p-card u-align-text--center js-os-select" style="cursor: pointer;" data-os="linux">
```

Comment:
A bit quick'n'dirty way to make this not look broken without removed `p-logo-link`. 

These pages don't seem linked from anywhere anymore, and are supposed to be removed as part of [WD-18896](https://warthogs.atlassian.net/browse/WD-18896) anyway.